pkg/cmd/cmd.go

```diff
@@ -112,3 +114,16 @@ func handleResponse(body []byte, contentType string, statusCode int, cmd *cobra.
 
 	return fmt.Errorf("request failed with status code %d\n", statusCode)
 }
+
+func openInBrowser(url string) error {
+	var err error
+	switch runtime.GOOS {
+	case "windows":
+		err = exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Run()
```

Comment:
That's some mastership in hackery 🙈

Reply:
I copied from https://github.com/efficientgo/e2e/blob/main/interactive/interactive.go#L24! 😅